package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "revanced-web-manager",
+	"version": "0.0.1",
```

Comment:
We will likely use semantic-release which does not support this initial version. I think the initial version has to be set to 1.0.0 which makes sense because the first release is a full release (v1).